terraform/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" {
+  value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "cd_webhook" {
+  value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
You should add `sensitive = true` to this output otherwise it gets logged in the Travis logs, so your app deployment password is leaked.

Reply:
Done